tests/app/dao/test_provider_details_dao.py

```diff
@@ -171,21 +171,20 @@ def set_primary_sms_provider(identifier):
 
 
 def test_can_get_sms_all_providers(restore_provider_details):
-    sms_providers = get_provider_details_by_notification_type('sms')
+    sms_providers = get_provider_details_by_notification_type('sms', False)
     assert len(sms_providers) == 6
```

Comment:
Why is this even here? Do we care that we have exactly 6 sms providers?

Reply:
It makes sense to me given the name of the test.  How else would we know we got all of them?

Reply:
Until one gets added.  I can see that a test that ensure some data gets returned when calling the method, but I disagree with validating that we got some precise number.

Reply:
I set this to `len(...) >= 1` so it's making sure something is there, but not a specific number